### ISSUE TITLE
Query about mat files

### ISSUE THREAD

**Issue by srdg** [ushasi/Earth-on-Canvas-dataset#3]
Hi, 
Could you please elaborate on how the mat files are structured?

I understand that the `wv_embeddings.mat` file contains a 300-d semantic information vector per each of the 14 classes.

I also understand that the `Imgmat` and `sketchmat` files respectively contain a feature against each sample. Could you inform from which network's output these features are taken? Please correct me if my assumption is wrong.

I am unfortunately unable to conclude anything regarding the `softlabels.mat` file. I see that it is a binary 1D vector of size 44100. Could you please provide a little more information about this file?

**Comment by ushasi** [comment]
Hi,
yes, (i) wv_embeddings.mat consists of the 300-d w2v embeddings; (ii) Imgmat corresponds to the fine-tuned features from an Imagenet pretrained model for the image data; (iii) Sketchmat corresponds to the sketch features similarly (if you need the codes for finetuning these datasets, you can use the following repository (https://github.com/ushasi/Fine-tuning-and-feature-extraction-from-pretrained-models); (iv) Softlabels consists of the actual class-labels of the image or sketch data. You can synthetically create them based on which class folder a particular image belongs to.
